follow/urls.py

```diff
@@ -15,6 +17,8 @@
     path('users/<slug:from_username>/accept/', view=accept_follow_request, name='accept_follow_request'),
     path('users/<slug:from_username>/reject/', view=reject_follow_request, name='reject_follow_request'),
     path('users/<slug:from_username>/unfollow/', view=unfollow_request, name='unfollow_request'),
+    path('remote/<path:url>/accept', view=accept_remote_follow_request, name='accept_remote_request'),
+    path('remote/<path:url>/reject', view=reject_remote_follow_request, name='reject_remote_request'),
```

Comment:
Can you add a view test for this, I think the accept will be consumed by the url (since path accepts '/'s), i.e. url could be <author_url>/accept instead of just <author_url>

Reply:
Working on it

Reply:
I think you are right, the accept will be consider as a part of url. Do you think swapping url's and accept's position will solve the problems?

Reply:
I'm thinking you could probably do this and it would work fine: 
```suggestion
    path('remote/follow/accept/<path:url>', view=accept_remote_follow_request, name='accept_remote_request'),
    path('remote/follow/reject/<path:url>', view=reject_remote_follow_request, name='reject_remote_request'),
```

Reply:
I'm currently working on a fix for this

Reply:
@armiantos updated routes and added a view test to verify